Resources/Prototypes/Entities/Clothing/Shoes/speedboots.yml

```diff
@@ -0,0 +1,45 @@
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingShoesSpeedBoots
+  name: speed boots
+  description: Allows you to run much faster!
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/speedboots.rsi
+    state: icon
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/speedboots.rsi
+  - type: NoSlip
+  - type: StaticPrice
+    price: 500
+  - type: ClothingSpeedModifier
+    walkModifier: 1
+    sprintModifier: 1.2
+    enabled: true
+  - type: Tag
+    tags:
+    - ClothingShoesSpeedBoots
+
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingShoesSpeedBootsClown
```

Comment:
Why are you not deriving this from ClothingShoesSpeedBoots, and only specifying the two or so differences?